vizro-core/src/vizro/models/_components/table.py

```diff
@@ -52,12 +55,32 @@ def __getitem__(self, arg_name: str):
         return self.figure[arg_name]
 
     @_log_call
+    def pre_build(self):
+        if self.actions:
+            kwargs = self.figure._arguments.copy()
```

Comment:
I decided to propagate them because they can be used in a `custom table function` in some way (eg `kwargs["abc"]`) and I don't want to compromise the proper operation of the function.